### PR TITLE
adding geometry glitches examples

### PR DESCRIPTION
**Pull request by afvc**

### CODE REVIEW

File: docs/components/geometry.md [hunk 1]
```diff
@@ -6,7 +6,23 @@ parent_section: docs
 section_title: Components
 section_order: 4
 source_code: src/components/geometry.js
-examples: []
+examples: 
+
+ - title:Creating Shapes
+    src: https://glitch.com/edit/#!/ex-2-geometry?path=index.html:1:0
```
Comment (cvan) [inline docs/components/geometry.md]: you can probably remove `?path=index.html:1:0` (or add them to the other URLs)

**Comment by cvan** [comment]
thanks for the contributions. these are nice 👍

File: docs/components/geometry.md (hunk 1 above)
Comment (ngokevin) [reply docs/components/geometry.md]: Nope, you need these so they don't route to the README.

**Comment by ngokevin** [comment]
Like https://github.com/aframevr/aframe/pull/3006, these require some work to format and make consistent. If @afvc is free, or whenever I have a bit of free time, we can get around to it.

File: docs/components/geometry.md (hunk 1 above)
Comment (cvan) [reply docs/components/geometry.md]: my question was more about the consistency with the URLs defined below

**Pull request merged**